python_modules/libraries/dagster-dbt/dagster_dbt/dbt_project.py

```diff
@@ -116,10 +116,10 @@ def __init__(
             raise DagsterDbtProjectNotFoundError(f"project_dir {project_dir} does not exist.")
 
         packaged_project_dir = Path(packaged_project_dir) if packaged_project_dir else None
-        if using_dagster_dev() and packaged_project_dir:
+
+        current_project_dir = project_dir
+        if packaged_project_dir and packaged_project_dir.exists():
```

Comment:
I had this gated by `using_dagster_dev` before to ensure that we were always pointed at the source of truth project directory and not a potentially out of sync package data copy.

Reply:
Hmm, shouldn't the logic then be:

```python
current_project_dir = project_dir
if not using_dagster_dev() and packaged_project_dir and packaged_project_dir.exists():
    current_project_dir = packaged_project_dir
```